packages/@lwc/engine-dom/src/renderer.ts

```diff
@@ -187,6 +187,15 @@ export function createComment(content: string): Node {
     return document.createComment(content);
 }
 
+export function cloneNode(node: Node, deep: boolean): Node {
+    return node.cloneNode(deep);
+}
+
+export function createFragment(html: string): Node | null {
+    // Not doing document.createFragment(html) because of IE11.
```

Comment:
I actually realized today that [there is no `document.createFragment(html)`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment)... I must have dreamt it. 🤦  Sorry for confusing things!